docs/UserGuide.adoc

```diff
@@ -115,13 +120,13 @@ Examples:
 
 * `find John` +
 Returns `john` and `John Doe`
-* `find Betsy Tim John` +
+* `search Betsy Tim John` +
 Returns any person having names `Betsy`, `Tim`, or `John`
 
-=== Deleting a person : `delete`
+=== Deleting a person : `delete or d`
```

Comment:
possible to set another aliase as "rm" because this is what we usually use on unix and linux